### DISCUSSION TITLE
[graphql-upload] Broken `Upload` type

### DISCUSSION THREAD

**Discussion by paulrostorp** [DefinitelyTyped/DefinitelyTyped#58120] [Issues with a @types package]
It seems https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57889 has broken the package:

![Screenshot 2022-01-11 at 19 00 57](https://user-images.githubusercontent.com/1866649/148996774-fbc6b804-2cd5-4bb1-8c53-d6438f46b9a7.png)

cc: @petarvujovic98

[a comment by an automated account is not shown]

**Comment by petarvujovic98** [comment]
Hello @paulrostorp, could you show me a use case where you manually instantiate the Upload class without the object already being created by the graphql-upload library through middleware?

**Reply by paulrostorp** [reply]
Hi,
I use it as part of my testing suite, to mock a file upload. There is likely another way to achieve this, but given that `Upload` is a class, they types should allow to instantiate it right ?
